src/frontend/components/SøknadsSteg/OmBarnetUtfyllende/OmBarnetUtfyllende.tsx

```diff
@@ -33,64 +32,56 @@ const InternKomponent: React.FC<{ barn: IBarn }> = ({ barn }) => {
-                                skjema.visFeilmeldinger
-                            )}
+                    <Undertittel>
+                        <SpråkTekst
+                            id={'ombarnet-utfyllende.institusjon.undertittel'}
+                            values={{ navn: barn.navn }}
                         />
-                    </>
-                )}
-                {skjema.felter.institusjonsnavn.erSynlig && (
-                    <Input
-                        label={
-                            <FormattedMessage
-                                id={
-                                    OmBarnetUfyllendeSpørsmålSpråkId[
-                                        OmBarnetUfyllendeSpørsmålsId.institusjonsadresse
-                                    ]
-                                }
-                            />
-                        }
-                        {...skjema.felter.institusjonsadresse.hentNavInputProps(
-                            skjema.visFeilmeldinger
-                        )}
-                    />
-                )}
-                {skjema.felter.institusjonsnavn.erSynlig && (
-                    <Input
-                        bredde={'S'}
-                        label={
-                            <FormattedMessage
-                                id={
-                                    OmBarnetUfyllendeSpørsmålSpråkId[
-                                        OmBarnetUfyllendeSpørsmålsId.institusjonspostnummer
-                                    ]
-                                }
-                            />
-                        }
-                        {...skjema.felter.institusjonspostnummer.hentNavInputProps(
-                            skjema.visFeilmeldinger
-                        )}
-                    />
+                    </Undertittel>
                 )}
+                <SkjemaFeltInput
+                    felt={skjema.felter.institusjonsnavn}
+                    visFeilmeldinger={skjema.visFeilmeldinger}
+                    labelSpråkTekstId={
+                        OmBarnetUfyllendeSpørsmålSpråkId[
+                            OmBarnetUfyllendeSpørsmålsId.institusjonsnavn
+                        ]
+                    }
+                />
+                <SkjemaFeltInput
+                    felt={skjema.felter.institusjonsadresse}
+                    visFeilmeldinger={skjema.visFeilmeldinger}
+                    labelSpråkTekstId={
+                        OmBarnetUfyllendeSpørsmålSpråkId[
+                            OmBarnetUfyllendeSpørsmålsId.institusjonsadresse
+                        ]
+                    }
+                />
+                <SkjemaFeltInput
+                    felt={skjema.felter.institusjonspostnummer}
+                    visFeilmeldinger={skjema.visFeilmeldinger}
+                    labelSpråkTekstId={
+                        OmBarnetUfyllendeSpørsmålSpråkId[
```

Comment:
Her skal du i teorien kunne bruke bare punktum, så får du opp riktig forslag i stedet for å hente ut via spørsmålsid :)

Reply:
Nice! Det ble ryddigere med en gang ja!